source/cli/appservices-deploy-configure.txt

```diff
@@ -0,0 +1,87 @@
+.. _appservices-deploy-configure:
+
+============================
+appservices deploy configure
+============================
+
+.. default-domain:: mongodb
```

Comment:
are we doing metadata for these pages?

Reply:
Nope! These pages are automatically generated, so we generally don't edit them. Any edits we make will be overwritten when they're regenerated.